internal/topology/variables/schema_test.go

```diff
@@ -186,6 +186,53 @@ func Test_convertToAPIExtensionsJSONSchemaProps(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "pass for schema validation with CEL validation rules",
```

Comment:
Is there a negative case that can be added here?

Reply:
I don't think so, conversions here are of compatible types and straight-forward conversion with rules, etc just strings.

Reply:
Agree